config/crds/kudo_v1alpha1_teststep.yaml

```diff
@@ -22,6 +22,38 @@ spec:
         delete:
           description: Objects to delete at the beginning of the test step.
           items:
+            properties:
```

Comment:
🤷‍♀ looks like nobody run `make manifests` for a long time?

Reply:
Yeah this should probably be part of our CI workflow to make sure there's no diffs here...